jdbc/src/main/scala/zio/sql/SqlDriverLiveModule.scala

```diff
@@ -45,7 +45,7 @@ trait SqlDriverLiveModule { self: Jdbc =>
 
     def read[A](read: Read[A]): Stream[Exception, A] =
       ZStream
-        .managed(pool.connection)
+        .scoped(pool.connection)
         .flatMap(readOn(read, _))
```

Comment:
Don't we need Stream.scoped() alternative that would accept the ZStream?
Just wondering what's the difference between
```
ZStream
        .scoped(pool.connection)
        .flatMap(readOn(read, _))
```
and 
```
ZStream
        .scoped(pool.connection.flatMap(readOn(read, _)))
```
Second alternative is currently not possible.

Reply:
There is `ZStream.unwrapScoped` which is equivalent to the previous `ZStream.unwrapManaged` and those were the only two operators for converting a `ZManaged` into a `ZStream` so I don't think there is any missing operator. If I understand the signatures correctly `pool.connection` is a resource so the second wouldn't compile because `connection is a `ZManaged` or `ZIO` and `readOn` returns a stream.